### DISCUSSION TITLE
How to continue a thread or get  a list of threads?

### DISCUSSION THREAD

**Discussion by JBlakeMCAA** [RageAgainstThePixel/com.openai.unity#253] [Q&A] [answered]
Ability to continue a conversation by getting a list of threads or storing  unique identifier in the metadata of the message?
Would like to show a list of Assistants and once selecting an assistant show a list of messages.
[Retrieve threadBeta](https://platform.openai.com/docs/api-reference/threads/getThread)
https://platform.openai.com/docs/api-reference/threads/getThread?lang=node.js

**Comment by StephenHodgson** [comment, marked as answer]
This isn't directly part of the open AI API. You'll need to store the value yourself and retrieve it later.
